examples/fci-wave-logn/fci-wave.cxx

```diff
@@ -17,7 +17,7 @@ class FCIwave : public PhysicsModel {
     Field3D f_B = f / Bxyz;
     
     f_B.splitYupYdown();
-    mesh->getParallelTransform().integrateYUpDown(f_B);
+    mesh->getParallelTransform().integrateYupYdown(f_B);
 
     // integrateYUpDown replaces all yup/down points, so the boundary conditions
```

Comment:
Small point: `integrateYUpDown` in comment probably should be updated too

Reply:
While you're changing it, should probably be `integrateParallelSlices` rather than `integrateYupYdown` now, since #1635.

Reply:
gcc recommended `integrateYupYdown` :smile: 
Unfortunately I don't know whether we can teach gcc to show a different function ...